src/harness/testcases/registration_testcase.py

```diff
@@ -147,3 +147,69 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_26(self):
+    """Category Error in Array request (responseCode 202)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE for the second, third, and fourth CBSDs.
+    """
+
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set up unique FCC IDs
```

Comment:
I have no strong opinions on the ordering so I'm happy with your proposal. Let's only rename to registration_* if we have to change the names for another reason (e.g. if we decided to re-order the files later); renaming at any point will cause headache for anyone who currently has a PR under review and I'd prefer to minimize that :)

Another thing to consider is how much we want to diversify the parameters for the devices: different locations, max EIRPs, etc. I don't have any strong opinions on it but it might be worth thinking about.